packages/ui/src/hooks/useIsMobile.ts

```diff
@@ -0,0 +1,6 @@
+import { useWindowDimensions } from 'tamagui'
+
+export default function useIsMobile() {
```

Comment:
nit: can we change this to something like `useIsWindowNarrow`? not all mobile clients match this, and narrow "desktop" clients (with cursor, etc) will (and should) match here – so I'm worried that this function could be misused in the future and require cleanup.

Reply:
good point, fixed!